adapters/appnexus/appnexus.go

```diff
@@ -82,7 +82,7 @@ type appnexusReqExt struct {
 var maxImpsPerReq = 10
 
 func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
-	memberIds := make(map[string]bool)
+	memberIds := make(map[string]struct{})
```

Comment:
nitpick - I think this should be declared where it is being used. Before for loop, perhaps?

Reply:
https://github.com/prebid/prebid-server/pull/2632/commits/0a16005e7f948517fd599b62c4f882d0334f3b6b makes the changes